src/librustc/metadata/filesearch.rs

```diff
@@ -295,7 +294,7 @@ fn find_libdir(sysroot: &Path) -> String {
 
 #[cfg(windows)]
 fn find_libdir(_sysroot: &Path) -> String {
-    "bin".to_string()
+    "lib".to_string()
```

Comment:
Do we need to do lib32/lib64 probing on Windows?  I suppose there's no harm in that though...